packages/runtime/lib/api.js

```diff
@@ -37,7 +43,7 @@ class RuntimeApi {
         }
       }
 
-      const app = new PlatformaticApp(service, loaderPort, logger, serviceTelemetryConfig, serverConfig, !!config.managementApi)
+      const app = new PlatformaticApp(service, loaderPort, logger, serviceTelemetryConfig, serverConfig, !!config.managementApi, metricsConfig)
```

Comment:
I think it's time to pass the whole `config` instead of passing `config.foo`, `config.bar`...